python-api-examples/speech-recognition-from-microphone.py

```diff
@@ -70,6 +73,66 @@ def get_args():
         help="Valid values are greedy_search and modified_beam_search",
     )
 
+    parser.add_argument(
+        "--num-threads",
```

Comment:
You did not use this argument, please delete it.

Reply:
Updated